android/library/src/main/res/drawable/default_button_repeatable_disabled_drawable.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+	<item>
+		<shape android:shape="oval">
+			<solid android:color="@color/lightGray"/>
+		</shape>
+	</item>
+	<item>
+		<shape android:shape="oval">
+			<solid android:color="#88FFFFFF"/>
```

Comment:
This color has to come from the theme. @color/mycolor

Reply:
Ok, I created two types for transparency, default and disabled for the theme